src/database/db_build.sql

```diff
@@ -1,5 +1,43 @@
 BEGIN;
 
+DROP TABLE IF EXISTS users, projects, donation;
```

Comment:
I think you should add `CASCADE` in this line

>to drop a table that is referenced by a view or a foreign-key constraint of another table, CASCADE must be specified. (CASCADE will remove a dependent view entirely, but in the foreign-key case it will only remove the foreign-key constraint, not the other table entirely.)
https://www.postgresql.org/docs/9.4/static/sql-droptable.html